src/app/clients/page.tsx

```diff
@@ -9,7 +9,7 @@ const dataFetch: () => Promise<Schema["clients"][] | null> = async () => {
 };
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data = await dataFetch();
+    const data: Datum[] = (await dataFetch()) ?? [];
```

Comment:
can you move the `?? []` to the `dataFetch`, so that it _can't_ return `null`?
e.g.
```tsx
const dataFetch = async (): Promise<Schema["clients"][]> => {
    // ...
    return response.data ?? [];
}
```

Reply:
resolved